app/src/main/java/com/mensinator/app/NotificationDialog.kt

```diff
@@ -0,0 +1,71 @@
+package com.mensinator.app
+
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.mensinator.app.ui.theme.MensinatorTheme
+
+@Composable
+fun NotificationDialog(
+    messageText: String,
+    onDismissRequest: () -> Unit,
+    onSave: (String) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    var newMessageText by remember { mutableStateOf(messageText) }
+
+    AlertDialog(
+        onDismissRequest = onDismissRequest,
+        modifier = modifier,
+        title = {
+            Text(text = stringResource(R.string.period_notification_message))
```

Comment:
@carstenhag I was passing in `title` as a parameter before your changes because I felt that `NotificationDialog` could be reused for the Ovulation Reminder? Let me know if that's unviable

Reply:
Ah I see. Imo it's not necessary to prepare for this already. It might never be implemented or it will have to behave in a different way. But yeah, would have left it if I had known your plan :D

Reply:
Haha I see, I was planning to give it a go after this gets merged, but I can refactor it then as you say :)